### ISSUE TITLE
cset error: failed to create shield

### ISSUE THREAD

**Issue by ufechner7** [lpechacek/cpuset#26]
When I execute the command:
```
sudo cset shield --cpu 1-3
```
I get the error message:
```
cset: --> failed to create shield, hint: do other cpusets exist?
cset: **> passed name=/user, which is not a string or CpuSet
```
Before executing this command I had the following list:
```
pi@navio:/sys/fs/cgroup/cpuset $ cset set -l
cset: 
         Name       CPUs-X    MEMs-X Tasks Subs Path
 ------------ ---------- - ------- - ----- ---- ----------
         root        0-3 y       0 y   114    0 /
```
Kernel version:
```
pi@navio:/sys/fs/cgroup/cpuset $ uname --all
Linux navio 4.4.36-a7765e7-emlid-v7+ #41 SMP PREEMPT Mon Mar 20 18:48:32 MSK 2017 armv7l GNU/Linux
```
Any idea how to fix this problem?

**Comment by lpechacek** [comment]
Thanks for the report, Uwe! This should be fixed by upcoming v1.6 release. If you are interested in testing the working version, it is sitting in `python3` branch.

The root cause is that `str` provided by `python-future` module is not the native Python `str` returned by `optparse`. This is resolved by commit 84af92e.

**Comment by lpechacek** [comment]
v1.6 is out. Either get updated packages from your distro vendor or download them at https://download.opensuse.org/repositories/home:/LPechacek:/cpuset-release/
